hack/boskos.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
```

Comment:
what is the intention of this script and where this will used/triggered?

Reply:
This script would be used to update the user-data of the Boskos resources. This should be used within a pod to be able to access the Boskos host. I'll add a comment on at the beginning.